packages/plugin-vue/package.json

```diff
@@ -35,7 +35,7 @@
   },
   "homepage": "https://github.com/vitejs/vite/tree/main/packages/plugin-vue#readme",
   "peerDependencies": {
-    "vite": "^3.0.0",
+    "vite": "^3.0.0 || ^4.0.0",
```

Comment:
I don't think we want to support 4.0.0 yet until it's stable, you can sset `--legacy-peer-deps` to work around the issue at the meantime.

Reply:
I think the unstable versions should support each other. If I'm on alpha Vite I would also want to use alpha Vite plugins right?

Reply:
Yeah you can, but instead of updating Vite to support it, you should relax the package manager to allow it first too, since we don't guarantee it would be compatible yet.